Flaky E2E test TestModelServingRollingUpdateMaxUnavailable

https://github.com/volcano-sh/kthena/actions/runs/22030492766/job/63654732261?pr=776
https://github.com/FAUST-BENCHOU/kthena/actions/runs/22031394532/job/63657053196?pr=18
```
=== RUN   TestModelServingRollingUpdateMaxUnavailable
    model_serving_test.go:430: Creating ModelServing with 4 replicas and maxUnavailable=2
    model_serving_test.go:435: Waiting for initial ModelServing (4 replicas) to be ready
    utils.go:33: Waiting for ModelServing to be ready...
    utils.go:39: Error getting ModelServing test-rolling-update-maxunavailable, retrying: client rate limiter Wait returned an error: context deadline exceeded
    utils.go:49: 
        	Error Trace:	/home/runner/work/kthena/kthena/test/e2e/utils/utils.go:49
        	            				/home/runner/work/kthena/kthena/test/e2e/controller-manager/model_serving_test.go:436
        	Error:      	Received unexpected error:
        	            	client rate limiter Wait returned an error: context deadline exceeded
        	Test:       	TestModelServingRollingUpdateMaxUnavailable
        	Messages:   	ModelServing did not become ready
--- FAIL: TestModelServingRollingUpdateMaxUnavailable (300.01s)
```

--- a/test/e2e/controller-manager/model_serving_test.go
+++ b/test/e2e/controller-manager/model_serving_test.go
@@ -408,7 +408,7 @@ func TestModelServingRollingUpdateMaxUnavailable(t *testing.T) {
 								Containers: []corev1.Container{
 									{
 										Name:  "test-container",
-										Image: "nginx:latest", // Initial image
+										Image: "nginx:1.28.2", // Initial image
 										Ports: []corev1.ContainerPort{
 											{
 												Name:          "http",
@@ -708,7 +708,7 @@ func createRole(name string, roleReplicas, workerReplicas int32) workload.Role {
 				Containers: []corev1.Container{
 					{
 						Name:  "test-container",
-						Image: "nginx:latest",
+						Image: "nginx:1.28.2",
 						Ports: []corev1.ContainerPort{
 							{
 								Name:          "http",
@@ -725,7 +725,7 @@ func createRole(name string, roleReplicas, workerReplicas int32) workload.Role {
 				Containers: []corev1.Container{
 					{
 						Name:  "worker-container",
-						Image: "nginx:latest",
+						Image: "nginx:1.28.2",
 						Ports: []corev1.ContainerPort{
 							{
 								Name:          "http",
@@ -752,7 +752,7 @@ func createBasicModelServing(name string, servingGroupReplicas int32, roles ...w
 						Containers: []corev1.Container{
 							{
 								Name:  "test-container",
-								Image: "nginx:latest",
+								Image: "nginx:1.28.2",
 								Ports: []corev1.ContainerPort{
 									{
 										Name:          "http",
@@ -801,7 +801,7 @@ func createInvalidModelServing() *workload.ModelServing {
 								Containers: []corev1.Container{
 									{
 										Name:  "test",
-										Image: "nginx:latest",
+										Image: "nginx:1.28.2",
 									},
 								},
 							},
@@ -845,7 +845,7 @@ func TestModelServingRollingUpdateMaxUnavailableWithBadImage(t *testing.T) {
 								Containers: []corev1.Container{
 									{
 										Name:  "test-container",
-										Image: "nginx:latest",
+										Image: "nginx:1.28.2",
 										Ports: []corev1.ContainerPort{
 											{
 												Name:          "http",
@@ -992,7 +992,7 @@ func TestLWSAPIBasic(t *testing.T) {
 						Containers: []corev1.Container{
 							{
 								Name:            "worker",
-								Image:           "nginx:latest",
+								Image:           "nginx:1.28.2",
 								ImagePullPolicy: corev1.PullIfNotPresent,
 								Ports: []corev1.ContainerPort{
 									{

